src/sui/themes/rtd-site/elements/button.overrides

```diff
@@ -0,0 +1,113 @@
+.ui.button {
+  vertical-align: middle;// Required for .button to be aligned with .button.animated
+}
+
+
+.ui.basic.buttons form > .button {
+  &:last-child {
+    border-top-right-radius: 0rem;
+    border-bottom-right-radius: 0rem;
+  }
+
+  &:hover {
+    box-shadow: none;
+  }
+
+  border-left: 0rem;
+}
+
+/*******************************
+     Gradient Variants (new)
+*******************************/
+
+/*-------------------
+      Secondary
+      Gradient
+--------------------*/
+
+.ui.secondary.buttons.gradient .button,
+.ui.secondary.button.gradient {
+  background-image: @secondaryGradientBackgroundImage;
+}
+.ui.secondary.buttons.gradient .button:hover,
+.ui.secondary.button.gradient:hover {
+  background-image: @secondaryGradientHoverBackgroundImage;
+}
+.ui.secondary.buttons .button:focus,
+.ui.secondary.button:focus {
+  background-image: @secondaryGradientFocusBackgroundImage;
+}
+.ui.secondary.buttons .active.button,
+.ui.secondary.buttons .active.button:active,
+.ui.secondary.active.button,
+.ui.secondary.button .active.button:active {
+  background-image: @secondaryGradientActiveBackgroundImage;
+}
+
+
+/*******************************
+     Outline Variants (new)
+*******************************/
+/* 
+  TODO1: Override .basic buttons with these styles and remove the .outline class.
+  TODO2: Add the full list of .basic buttons to our /examples/theme/index.html.
+*/
+
+.ui.outline.button {
+  background: transparent none;
+  text-shadow: none !important;
+  transition: box-shadow @defaultDuration;
+}
+/* Mixin for .outline states */
+.buttonOutlineState(@color) {
+  box-shadow: 0px 0px 0px 1px @color inset !important;
+  color: @color;
+
+  &:hover,
+  &:focus,
+  &.active,
+  &.active:focus {
+    box-shadow: 0px 0px 0px 2px @color inset !important;
+    background: none !important;
+  }
+}
+
+/* States */
+.ui.outline.button {
+
+  &.black     { .buttonOutlineState(@black) }
+  &.primary   { .buttonOutlineState(@primaryColor) }
+  &.secondary { .buttonOutlineState(@secondaryColor) }
+  &.blue      { .buttonOutlineState(@primaryColor) }
+  &.green      { .buttonOutlineState(@green) }
+
+}
+
+/*******************************
+        Size Variants
+*******************************/
+.ui.tiny.button {
+  padding: (@tinyVerticalPadding + .05) @tinyHorizontalPadding @tinyVerticalPadding;
+  font-weight: @normal;
+}
+.ui.mini.button {
+  padding: (@miniVerticalPadding + .05) @miniHorizontalPadding @miniVerticalPadding;
+  font-weight: @normal;
+}
+
+/*******************************
+      Group Wrapper (new)
+*******************************/
+.group.wrapper {
+  .ui.button {
+    margin-bottom: @horizontalMargin;
+  }
+  .ui.button.large {
+    margin-right: 6px;
+    margin-bottom: 8px;
+  }
+}
+/* behavior with other elements */
+.text.lead + .group.wrapper {
+  margin-top: 2em;
+}
```

Comment:
I'm using it here:
https://github.com/readthedocs/site-community/blob/0f4c9b83ebd22a7c488ba1055cd692116ad24453/examples/mockups/index.html#L53-L63

I didn't find any SUI native options for styling elements' spacings.
I styled the selector this way because I think its something useful that we'll end up reusing multiple times in different instances over the marketing pages.

Reply:
Already noted somewhere else, but with some of the changes to the `.text.lead` that I mentioned, this wouldn't be necessary anymore. A segment would increase the font size and provide bottom border, as would a header.

Reply:
Done. Removed the  `.group.wrapper` and `.text.lead` in favor of SUI native options.